Tabloid/client/src/components/TagManagment.js

```diff
@@ -0,0 +1,16 @@
+const baseUrl = 'api/Tag';
+
+export const getAllTags = () => {
+  return fetch(baseUrl)
+    .then((res) => res.json())
+};
+
+export const addTag = (tag) => {
```

Comment:
is addTag supported by your API yet? if not, remove this function from this PR.

Reply:
@heymonicakay got rid of add tax on the tag mangment.js